src/triage/component/postmodeling/contrast/evaluator.py

```diff
@@ -0,0 +1,543 @@
+"""
+Model Evaluator
+
+This script contain a set of elements to help the postmodeling evaluation
+of audited models by triage.Audition. This will be a continuing list of
+routines that can be scaled up and grow according to the needs of the
+project, or other postmodeling approaches.
+
+To run most of this routines you will need:
+    - S3 credentials (if used) or specify the path to both feature and
+    prediction matrices.
+    - Working database db_engine.
+"""
+
+import pandas as pd
+import numpy as np
+from sqlalchemy.sql import text
+from matplotlib import pyplot as plt
+from sklearn import metrics
+from collections import namedtuple
+
+from utils.file_helpers import download_s3
+from utils.test_conn import db_engine
+from utils.aux_funcs import recombine_categorical
+
+# Get individual model_ids from Audition outcome
+
+
+def get_models_ids(audited_model_group_ids):
```

Comment:
Why is this function outside the class?

Reply:
Because is a utility function to get the `model_id's` out of `model_group_id`. I guess this should be in the utils directory, instead.